cve_bin_tool/cvedb.py

```diff
@@ -387,7 +384,9 @@ def populate_db(self):
                 # affected {Vendor(s), Product(s), Version(s)}
                 CVE = dict()
                 CVE["ID"] = cve_item["cve"]["CVE_data_meta"]["ID"]
-
+                CVE["summary"] = cve_item["cve"]["description"]["description_data"][0][
```

Comment:
Is there a reason we're using different language than NVD does?  (as in, calling this "summary" instead of "description")  After my snafu with product versus project in csv2cve, I'm thinking we should try to use the same terminology as NVD does when we can, but if there's a strong reason to switch to summary I'm willing to be convinced.

Reply:
As such, there is no particular reason for using summary. So I'll fix this.